src/main/webapp/app/overview/postings/answer-post/answer-post.component.ts

```diff
@@ -31,9 +31,33 @@ export class AnswerPostComponent implements OnInit {
     EditorMode = EditorMode;
     courseId: number;
 
-    // Only allow certain html tags and no attributes
-    allowedHtmlTags: string[] = ['a', 'b', 'strong', 'i', 'em', 'mark', 'small', 'del', 'ins', 'sub', 'sup', 'p'];
```

Comment:
@krusche , is there any reason why we where so restrictive with html tags. This made most of the markdown being lost while converting into html. IMO the added tags and attributes are not critical w.r.t. security... am I wrong?